langs/java.go

```diff
@@ -174,8 +177,15 @@ func mavenOpts() string {
 /*    TODO temporarily generate maven project boilerplate from hardcoded values.
 Will eventually move to using a maven archetype.
 */
-func pomFileContent(APIversion, javaVersion string) string {
-	return fmt.Sprintf(pomFile, APIversion, javaVersion, javaVersion)
+func pomFileContent(APIversion, javaVersion, groupId, pomType string) string {
+	if groupId == "io.fnproject.com" {
```

Comment:
Could this be `if groupId == "io.fnproject.com" || pomType == "maven"`?
The `Sprintf` seems identical in both cases.

Reply:
Done